lightwood/api/data_source.py

```diff
@@ -202,22 +199,35 @@ def __getitem__(self, idx):
 
         if not self.disable_cache:
             self.transformed_cache[idx] = sample
-            return self.transformed_cache[idx]
-        else:
-            return sample
+        return sample
 
     def get_column_original_data(self, column_name):
-        """
-
-        :param column_name:
-        :return:
-        """
         if column_name not in self.data_frame:
             nr_rows = self.data_frame.shape[0]
             return [None] * nr_rows
 
         return self.data_frame[column_name].tolist()
 
+    def lookup_encoder_class(self, column_type):
+        path = 'lightwood.encoders.{type}'.format(type=column_type)
+        module = importlib.import_module(path)
```

Comment:
This is declared here but bellow `importlib.import_module(path)` is used again, use this one bellow instead (unless you change the code as suggested, in which case we could remove this whole function actually and just replace it with:

`importlib.import_module(f'lightwood.encoders.{column_type}').default`